[Soulsync] broken update via update command

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Soulsync

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 11

### 📈 Which Proxmox version are you on?

9.1

### 📝 Provide a clear and concise description of the issue.

after running "update" the script fails to setup the venv since it can not find pip.  this happened from 2.4 to 2.5, and also again from 2.5 to 2.5.1 

subsequent  running "update" doesnt do anything since it detects the updated version and does not run again. 

### 🔄 Steps to reproduce the issue.

1   install soulsync in new lxc 
2  run update 



### ❌ Paste the full error output (if available).

```
  ✔️   Update available: soulsync 2.5.0 → 2.5.1
  ✔️   Stopped Service
  ✔️   Backed up Data
  ✔️   Deployed: soulsync (2.5.1)
  ⏳   Updating Python DependenciesUsing CPython 3.11.15
Creating virtual environment at: .venv
Activate with: source .venv/bin/activate
environment: line 47: /opt/soulsync/.venv/bin/pip: No such file or directory
  ✖️   in line 47: exit code 127 (Command not found): while executing command  /opt/soulsync/.venv/bin/pip install -r requirements.txt

  --- Last 20 lines of log ---
[2026-05-13 10:29:21] [INFO] Checking for update: soulsync
[2026-05-13 10:29:22] [OK] Update available: soulsync 2.5.0 → 2.5.1
[2026-05-13 10:29:22] [INFO] Stopping Service
[2026-05-13 10:29:24] [OK] Stopped Service
[2026-05-13 10:29:24] [INFO] Backing up Data
[2026-05-13 10:29:24] [OK] Backed up Data
[2026-05-13 10:29:24] [INFO] Fetching GitHub release: soulsync (2.5.1)
[2026-05-13 10:29:26] [OK] Deployed: soulsync (2.5.1)
[2026-05-13 10:29:26] [INFO] Updating Python Dependencies
[2026-05-13 10:29:27] [ERROR] in line 47: exit code 127 (Command not found): while executing command  /opt/soulsync/.venv/bin/pip install -r requirements.txt
  -----------------------------------

 📋 Full log: /tmp/create-lxc-f5589d70.log
```

### 🖼️ Additional context (optional).

steps to fix (still missing redeploy settings  though) 


cd /opt/soulsync
source .venv/bin/activate
python -m ensurepip
python -m pip install -r requirements.txt
systemctl restart  soulsync

--- a/ct/soulsync.sh
+++ b/ct/soulsync.sh
@@ -44,7 +44,7 @@ function update_script() {
     msg_info "Updating Python Dependencies"
     cd /opt/soulsync
     $STD uv venv --clear /opt/soulsync/.venv --python 3.11
-    $STD /opt/soulsync/.venv/bin/pip install -r requirements.txt
+    $STD uv pip install -r requirements.txt
     msg_ok "Updated Python Dependencies"
 
     mv /opt/soulsync-config.bak /opt/soulsync/config

